Dockerfile

```diff
@@ -29,9 +29,22 @@ FROM alpine:3.13.0
 # OpenSSL is required so wget can query HTTPS endpoints for health checking.
 RUN apk add --no-cache --update ca-certificates openssl
 
+ARG TARGETOS
+ARG TARGETARCH
+ARG TARGETVARIANT
+ARG GOMPLATE_VERSION=v3.9.0
+
+RUN wget -O /usr/local/bin/gomplate \
+  "https://github.com/hairyhenderson/gomplate/releases/download/${GOMPLATE_VERSION}/gomplate_${TARGETOS:-linux}-${TARGETARCH:-amd64}${TARGETVARIANT}" \
+  && chmod +x /usr/local/bin/gomplate
+
```

Comment:
I meant a completely separate build stage (before the final stage), so it can be cached separately from all the other stages (not that we utilize image cache today, but I definitely want to)